sbi/inference/base.py

```diff
@@ -47,7 +47,7 @@ def infer(
     init_kwargs: Optional[Dict] = None,
     train_kwargs: Optional[Dict] = None,
     build_posterior_kwargs: Optional[Dict] = None,
-) -> NeuralPosterior:
+) -> NeuralPotentialPosterior:
```

Comment:
should we remove this altogether? We removed it from the tutorials already, we don't want people to use this function anymore?

Reply:
I would be in favor of removing this, yes.

Reply:
solved: ScorePosterior is on the same level as Direct and others.